lib/sportradar/api.rb

```diff
@@ -48,12 +49,29 @@
 require "sportradar/api/nfl/scoring"
 require "sportradar/api/nfl/season"
 require "sportradar/api/nfl/situation"
+require "sportradar/api/nfl/stat_pack"
 require "sportradar/api/nfl/statistic"
 require "sportradar/api/nfl/summary"
 require "sportradar/api/nfl/team"
 require "sportradar/api/nfl/venue"
 require "sportradar/api/nfl/week"
 
+require "sportradar/api/nfl/stat"
```

Comment:
This guy may be pulled out if we don't end up using the `stat` class I pointed out in the other comment